### ISSUE TITLE
Mark messages as unread or bookmark them

### ISSUE THREAD

**Issue by Dominik-K** [gitterHQ/gitter#1688]
It would be nice to mark all messages up to a point in history as unread. I would use it as a form of a bookmark to remind me to read them. 

If there would be bookmark feature, that would fulfill my use case as well.

#1504 is somehow related in what @ktonga wants to achieve.

**Comment by MadLittleMods** [comment]
@Dominik-K You can already bookmark messages by their permalink.

 - Click a timestamp in the top-right of every message and bookmark the page
 - Alt + Click on a timestamp to get a markdown pointer in the chat input

**Comment by mydigitalself** [comment]
@MadLittleMods I don't think that bookmarking you describe is what is meant by the request. It's more a way to star/flag for follow up so you get a list somewhere of messages that are interesting/important/etc...

**Comment by MadLittleMods** [comment]
@mydigitalself Thanks for the clarification, related to https://github.com/gitterHQ/gitter/issues/365 in that case

Previous issue on this subject, https://github.com/gitterHQ/gitter/issues/516
